rules/windows/execution_initial_access_via_msc_file.toml

```diff
@@ -0,0 +1,98 @@
+Identifies the execution of a child process from a Microsoft Common Console file. Adversaries may embed a malicious command
+in an MSC file in order to trick victims into executing malicious commands.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Unusual Execution via Microsoft Common Console File"
+note = """## Triage and analysis
+
+### Investigating Execution via Microsoft Common Console File
+
+- Investigate the source of the MSC file.
+- Investigate the process execution chain (all spawned child processes and their descendants).
+- Investigate the process and it's descendants network and file events.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Contact the account owner and confirm whether they are aware of this activity.
+
+### Response and remediation
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- If the triage identified malware, search the environment for additional compromised hosts.
+  - Implement temporary network rules, procedures, and segmentation to contain the malware.
+  - Stop suspicious processes.
+  - Immediately block the identified indicators of compromise (IoCs).
+  - Inspect the affected systems for additional malware backdoors like reverse shells, reverse proxies, or droppers that attackers could use to reinfect the system.
+- Remove and block malicious artifacts identified during triage.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
+"""
+references = ["https://www.genians.co.kr/blog/threat_intelligence/facebook"]
+risk_score = 73
+rule_id = "e760c72b-bb1f-44f0-9f0d-37d51744ee75"
+severity = "high"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Tactic: Initial Access",  "Data Source: Windows", "Resources: Investigation Guide", "Data Source: Elastic Endgame", "Data Source: Elastic Defend"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+ process.parent.executable : "?:\\Windows\\System32\\mmc.exe" and endswith~(process.parent.args, "*.msc") and
```

Comment:
```suggestion
 process.parent.executable : "?:\\Windows\\System32\\mmc.exe" and endswith~(process.parent.args, ".msc") and
```

Reply:
I assume that wildcard is unnecessary, as `endswith` already matches everything prior to `.msc`?

Reply:
yes!